### ISSUE TITLE
Schema Visualizer

### ISSUE THREAD

**Issue by dallenbaldwin** [apollographql/apollo-studio-community#226]
## Context

I have a few lay users in Apollo Studio and the desire came up the other day to see the whole graph in a more visual manner. Our graph is growing (almost 100 entities now) and new, and even veteran, users can find it difficult to understand how everything works together; especially since ...interesting... decisions were made long ago with respect to entity names. 

The Schema page has the reference, but it seems much more tailored for a drill down versus overview

I was able to find an online tool called [GraphQL Voyager](https://ivangoncharov.github.io/graphql-voyager/) that helps with this, and feel it would be super nice to have something like it in Apollo Studio as well, possibly a new tab in the Schema page

## What type of feedback is this?

- [ ] Question
- [x] Idea
- [ ] Bug
- [ ] New example or operation collection

## Is there anything else you'd like to share about the Platform API?

We love Apollo Studio. It's the best GraphQL tool I've found to date and is partially responsible for decision makers above me being excited for, and more understanding of, our application's graph

**Comment by Y-Guo** [comment]
Hi @dallenbaldwin, I'm happy to share that [schema visualization](https://www.apollographql.com/blog/visualizing-federated-graphql-apis-with-graphos-studio) is now supported in studio! You should be able to access this under the Visualization tab on the Schema page 🚀

**Comment by piyushyadav1617** [comment]
Hey @Y-Guo I cannot see the Visualization tab in the studio, do I need to configer something to get the tab or is it available only after logging in?
